### PR TITLE
Improve Releasing during v0.11.7

### PR DESCRIPTION
**Pull request by jedevc**

### CODE REVIEW

**Comment by jedevc** [comment]
Oh no, we see a *lot* of hangs here: https://github.com/dagger/dagger/actions/runs/9469086782/job/26086942090?pr=7616

**Comment by gerhard** [comment]
Just noticed Rust SDK run hanging: https://github.com/dagger/dagger/actions/runs/9486637015/job/26141460197?pr=7642

image

There is no `rust test` trace in Dagger Cloud, only `rust lint` cc @vito 

image

image

Re-running.

**Comment by gerhard** [comment]
Re-running it fixed that run, but we have a different hang in this one: https://github.com/dagger/dagger/actions/runs/9486637029/job/26141460284?pr=7642

image

Same issue as before, we are not seeing the trace in Dagger Cloud.

**Comment by vito** [comment]
@gerhard The traces are probably being sent to Cloud but don't show up in Cloud because there's a known issue with the grouping logic: it doesn't show "running" traces, and these ones are "running" forever because of the hang

**Comment by gerhard** [comment]
Got it!

We are on `Attempt #2` and are still seeing the same hanging behaviour. The tests have finished, but the job is not completing and it will most probably timeout again. Will check the Engine logs next & share here too. Is there anything else that we can do to debug this?

**Comment by jedevc** [comment]
@gerhard this sounds like the issue we are currently working on debugging in discord - https://discord.com/channels/707636530424053791/1250122361131499664/1250466959594750022

**Comment by gerhard** [comment]
@jedevc thank you, will follow that conversation.

File: ci/go.mod [hunk 1]
```diff
@@ -2,49 +2,49 @@ module github.com/dagger/dagger/ci
 
 go 1.21.7
 
-require github.com/dagger/dagger/engine/distconsts v0.11.6
+require github.com/dagger/dagger/engine/distconsts v0.11.7
```
Comment (gerhard) [inline ci/go.mod:5]: Note to self: bump to `v0.11.8-rc.3` after we produce it.

File: go.mod [hunk 2]
```diff
@@ -3,8 +3,8 @@ module github.com/dagger/dagger
 go 1.21.7
 
 require (
-	dagger.io/dagger v0.11.6
-	github.com/dagger/dagger/engine/distconsts v0.11.6
+	dagger.io/dagger v0.11.8-rc.2
+	github.com/dagger/dagger/engine/distconsts v0.11.7
```
Comment (gerhard) [inline go.mod:7]: Note to self: bump to `v0.11.8-rc.3` after we produce it.

**Comment by gerhard** [comment]
Are you OK to rebase this @jedevc or should I?

I am keen to get this in so that we start collecting metrics on `0.11.7` behaviour in our CI.

**Comment by jedevc** [comment]
@gerhard rebased.

**Comment by gerhard** [comment]
Not sure what is going on here, this seems like a new issue to me: https://github.com/dagger/dagger/actions/runs/9519543834/job/26242956903?pr=7616#step:4:64

image

Re-running so that we can merge when all checks are green.

**Comment by jedevc** [comment]
@gerhard those errors come from the setup-go action - https://github.com/actions/setup-go/issues/424: they're non fatal, but yes, annoying.

**Comment by jedevc** [comment]
I *think* the `test` timeout is expected - I'm pretty sure this is the same `testdev` hang we saw before merging https://github.com/dagger/dagger/pull/7660.

**Comment by jedevc** [comment]
@gerhard while we wait on https://github.com/dagger/dagger.io/pull/3798, I just held back `test` to v0.11.6 - it should only be temporary, we can upgrade it to v0.11.8 later.

**Review by gerhard — APPROVED** [review]
OK, this workaround will have to do for now. The fix with `v0.11.8` should be coming in hot, hopefully today 🤞

**Pull request merged**